silta/silta.yml

```diff
@@ -12,14 +12,14 @@ php:
     DRUPAL_CLIENT_SECRET: drupal_client_secret_not_secure_used_only_locally
     DRUPAL_CLIENT_ID: drupal-client-id
   hashsalt: notsosecurehashnotsosecurehashnotsosecurehash
-  # This section is used in the template only, it will reinstall the site and apply
-  # all standard configuration at each commit. When creating a new project from this template
-  # you have to remove this section.
   postupgrade:
     afterCommand: |
       drush si minimal -y
       chmod +x /app/web/core/scripts/drupal
       cd /app/web && core/scripts/drupal recipe ../recipes/wunder_next_setup
+      drush cr
+      drush simple-oauth:generate-keys /app/oauth
```

Comment:
This won't work as the keys are generated on a running container storing the keys only to that containers ephemeral storage instead of storing them to the actual container image so they will be lost after the container is recreated (also not present on drupal pod as this is being run on shell pod).

Reply:
ah, that explains it :)